src/llama.cpp

```diff
@@ -21734,6 +21734,15 @@ static int32_t llama_chat_apply_template_internal(
         if (add_ass) {
             ss << "[|assistant|]";
         }
+    } else if (tmpl == "rwkv-world" || tmpl_contains("rwkv-world") || tmpl_contains("'User: ' + message['content'] + '\n\nAssistant:'")) {
+        for (auto message : chat) {
+            std::string role(message->role);
+            if (role == "user") {
+                ss << "User: " << message->content << "\n\nAssistant:";
```

Comment:
But do we need to add a space after `Assistant:` ? We currently have a space after `User: `

Reply:
From what I knew, it's not required according to @BlinkDL 's demo code.

Reply:
https://github.com/BlinkDL/ChatRWKV/blob/d67bbb34aeb01bff68a02aecb9721d992555dcea/API_DEMO_CHAT.py#L105